benchmark/python/sparse/dot.py

```diff
@@ -337,27 +340,41 @@ def print_benchmark_info(lhs, rhs, lhs_trans, fw):
 
     def run_benchmark(ctx=None, lhs="csr", lhs_trans=False, rhs="dns", fw="mxnet", rhs_density=1,
                       distribution="uniform"):
-        if lhs != "csr":
-            raise ValueError("Value other than csr for lhs not supported")
+
         if rhs_density > 1 or rhs_density < 0:
-            raise ValueError("rhs_density has to be between 0 and 1")
+            raise ValueError("Value other than csr for lhs not supported")
```

Comment:
The check and the error statement don't seem to match?

Reply:
Good catch! Fixed.